cognite/neat/utils/utils.py

```diff
@@ -27,6 +27,11 @@ def get_cognite_client_from_config(config: ServiceCogniteClient) -> CogniteClien
     return _get_cognite_client(config, credentials)
 
 
+def get_cognite_client_from_token(config: ServiceCogniteClient) -> CogniteClient:
+    credentials = Token(config.client_secret)
```

Comment:
Ideally we should change the config to explicitly state which how the client should be created.